sys/net/sock/Kconfig

```diff
@@ -28,4 +28,12 @@ config SOCK_URLPATH_MAXLEN
     help
         This value is used in sock_urlsplit().
 
+config DTLS_HANDSHAKE_BUFSIZE_EXP
+    int "Exponent for the DTLS buffer size (resulting in the buffer size 2^n)"
+    default 8
+    help
+        As the buffer size ALWAYS needs to be power of two, this option
+        represents the exponent of 2^n, which will be used as the size of the
+        buffer. The buffer is used to hold credentials during DTLS handshakes.
+
```

Comment:
I think this should be inside a section dedicated to Sock DTLS, which would depend on `USEMODULE_SOCK_DTLS`

Reply:
Makes sense.